tests/backup_restore_test.go

```diff
@@ -138,98 +178,127 @@ func TestPITR(t *testing.T) {
 		a.NoError(err)
 	}
 
-	_, stop := mysql.SetupTestInstance(t, port)
-	defer stop()
+	// test cases
+	t.Run("Buggy Application", func(t *testing.T) {
+		t.Parallel()
 
-	db, err := sql.Open("mysql", fmt.Sprintf("%s@tcp(%s:%d)/mysql", username, localhost, port))
-	a.NoError(err)
-	defer db.Close()
+		t.Log("[<t0] initialize database")
```

Comment:
to0 is a bit hard to understand.. I think initialize database and following logs are already descriptive.